packages/react/src/screens/core/User.tsx

```diff
@@ -0,0 +1,161 @@
+import { useCorbado } from '@corbado/react-sdk';
+import type { CorbadoUser, Identifier } from '@corbado/types';
+import type { FC } from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { PasskeyListErrorBoundary } from '../../components';
+import InputField from '../../components/ui2/input/InputField';
+import { PhoneInputField } from '../../components/ui2/input/PhoneInputField';
+import { LoadingSpinner } from '../../components/ui2/LoadingSpinner';
+import { Text } from '../../components/ui2/typography/Text';
+
+interface ProcessedUser {
+  name: string;
+  username?: string;
+  emails: Identifier[];
+  phoneNumbers: Identifier[];
+}
+
+export const User: FC = () => {
+  const { corbadoApp, isAuthenticated, globalError } = useCorbado();
+  const { t } = useTranslation('translation', { keyPrefix: 'user' });
+  const [currentUser, setCurrentUser] = useState<CorbadoUser | undefined>();
+  const [loading, setLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (!isAuthenticated) {
+      return;
+    }
+
+    void getCurrentUser();
+  }, [isAuthenticated]);
+
+  const headerText = useMemo(() => t('header'), [t]);
+  const nameFieldLabel = useMemo(() => t('name'), [t]);
+  const usernameFieldLabel = useMemo(() => t('username'), [t]);
+  const emailFieldLabel = useMemo(() => t('email'), [t]);
+  const phoneFieldLabel = useMemo(() => t('phone'), [t]);
+  const verifiedText = useMemo(() => t('verified'), [t]);
+  const unverifiedText = useMemo(() => t('unverified'), [t]);
+  const processUser = useMemo((): ProcessedUser => {
+    if (!currentUser) {
+      return {
+        name: '',
+        emails: [],
+        phoneNumbers: [],
+      };
+    }
+
+    return {
+      name: currentUser.fullName,
+      username: currentUser.identifiers.find(id => id.type === 'username')?.value,
+      emails: currentUser.identifiers.filter(id => id.type === 'email'),
+      phoneNumbers: currentUser.identifiers.filter(id => id.type === 'phone'),
+    };
+  }, [currentUser]);
+
+  const getCurrentUser = useCallback(async () => {
+    setLoading(true);
+    const result = await corbadoApp?.sessionService.getFullUser();
```

Comment:
We should get this through the useCorbado hook (see the 3rd TODO in the ticket for this PR). The rationale behind having this inside the react context is this: External developers might want to access this information themselves (e.g. because they want to get a user's email addresses or fullName).

Even if we don't expose most of the component related functionalities about the auth process itself to the developers (because it is too complicated) it makes sense in this case to provide developers with that data.